### PR TITLE
Moving in some simple Kafka channel case

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

* moving test cases to a root folder
* adding a simple kafka test case

NEXT (different PR)
* IM/Kafka channel used via broker/trigger
* some doc for each test case (simple background info)

### CODE REVIEW

**Review by slinkydeveloper — COMMENTED** [review]
Can you add a small doc in the root README for each test case? Be explicit about the params to tweak (e.g. numPartitions in kafka case defines the consumer parallelism), it might help

File: testcases/kc-test-config/100-channel.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: messaging.knative.dev/v1beta1
+kind: KafkaChannel
+metadata:
+  name: kafkatestchannel
+spec:
+  numPartitions: 3
+  replicationFactor: 3
```
[the remark this answers is not in the document]
Comment (matzew) [reply testcases/kc-test-config/100-channel.yaml:7]: yeah, the replication has also relevance on the setup of kafka.

We can start with default.  and have an advanced testcase too  - good point

**Comment by matzew** [comment]
@slinkydeveloper I've updated this.

I will add broker configs in a different PR...

**Review by slinkydeveloper — APPROVED** [review]

**Pull request merged**